umake/frameworks/ide.py

```diff
@@ -77,7 +77,6 @@ def __init__(self, *args, **kwargs):
             kwargs["required_files_path"] = current_required_files_path
         download_page = 'https://www.eclipse.org/downloads/'
         kwargs["download_page"] = download_page
-        kwargs["exec_rel_path"] = "eclipse"
```

Comment:
I still find it being mentioned twice in baseinstaller.py:
```umake/frameworks/baseinstaller.py:        self.exec_rel_path = kwargs.get("exec_rel_path", None)
umake/frameworks/baseinstaller.py:                          "match_last_link", "exec_rel_path"]:
```

Did you remove them and I didn't notice it? If we are going to remove all of them, better to ensure it's not anywhere anymore :)